dockerfiles/docker-entrypoint.sh

```diff
@@ -24,8 +24,8 @@ if [ -z "${BIND_ADDRESS}" ]; then
     export BIND_ADDRESS="${DEFAULT_BIND_ADDRESS}"
 fi
 
-export UWSGI_SETTINGS_PATH=/etc/searx/uwsgi.ini
```

Comment:
Why is it needed to set the default values here as well if they had been set in the `Dockerfile` at an earlier point of execution?

Reply:
It's not necessary at all, but:

* There may exist users who currently rely on the value being set here in previous versions - this prevents that from breaking for such users

* Laziness - removes ned to check for value being set or not 
But if you prefer I'm happy to remove it to just have the default set in one place

Reply:
Please remove it. If anyone complains about it, I am willing to take the blame. :D

Reply:
@kvch removed!